Specs/createScene.js

```diff
@@ -20,7 +20,7 @@ define([
         options = defaultValue(options, {});
 
         // save the canvas so we don't try to clone an HTMLCanvasElement
-        var canvas = defined(options.canvas) ? options.canvas : createCanvas();
+        var canvas = defined(options.canvas) ? options.canvas : createCanvas(10,10);
```

Comment:
Instead of the default of (10, 10) here, `PickSpec` should instead send in a 10x10 canvas.